dev/inc/tracker_ref.h

```diff
@@ -3,6 +3,8 @@
 
 #pragma once
 
+#include "SharedHelpers.h"
```

Comment:
why do you need include this header?

Reply:
Because SharedHelpers is used below. New compiler must be more sensitive to the symbols from that header being available before the tracker_ref template is stamped out. Today it's probably letting the resolution be more lazy.